README.md

```diff
@@ -1,36 +1,59 @@
 # Utils
 
-[![Build Status](https://travis-ci.org/kubernetes/utils.svg?branch=master)](https://travis-ci.org/kubernetes/utils)
+[![Build Status]](https://travis-ci.org/kubernetes/utils)
 
-Utils is a set of golang libraries that are not specific to
-Kubernetes. They should be available and useful to any other Go project
-out there.
+A set of Go libraries that provide low-level,
+kubernetes-independent packages supplementing the [Go
+standard libs].
 
 ## Purpose
 
-The Kubernetes project uses a lot of Golang patterns that are re-used in many
-difference places, and as the project is further split into different
-repositories, this repository is the perfect place to hold this common code.
+As Kubernetes grows and spins functionality out of its
+[core] and into cooperating repositories like
+[apiserver], [kubectl], [kubeadm], etc., the need
+arises for leaf repositories to house shared code and
+avoid cycles in repository relationships.
 
-## Goals
+This repository is intended to hold shared utilities
+with no Kubernetes dependence that may be of interest
+to any Go project.  See these [instructions for moving]
+an existing package to this repository.
 
-Go libraries in this repository must be:
+A proposed sibling repository, with the proposed name [common],
+will hold shared utilities that may depend on both this
+repository and other near-leaf repositories like [api].
```

Comment:
My understanding was that `common` would only depend on `utils`, with the only difference being that it would be "kubernetes specific"